components/Form/SummaryList/index.js

```diff
@@ -0,0 +1,21 @@
+const SummaryList = ({ name, listObject }) => (
```

Comment:
Might just be me, but wondering if there is a better name for `listObject`? Maybe something like `entries` as in "entires in the summary list". I'm not sure I'd know what `listObject` did without looking into the component.